### PR TITLE
feat: created different TableCell components

### LINKED ISSUE #166: feat: create different table cells components
### Describe the feature you'd like to request

We have different tables across the app. Somme cells of those tables are rewritten across the codebase and should be extracted to components.

### Describe the solution you'd like to see

For example, in the component [collection-items-data-list-view.tsx](https://github.com/ArkProjectNFTs/ark-market/blob/main/apps/arkmarket/src/app/collection/%5BcollectionAddress%5D/components/collection-items-data-list-view.tsx), this piece of code inside of a `TableCell`:
```tsx            
                <Link
                  prefetch={false}
                  href={`/token/${token.collection_address}/${token.token_id}`}
                  className="flex items-center gap-4"
                >
                  <Media
                    src={token.metadata?.image}
                    mediaKey={token.metadata?.image_key}
                    alt={token.metadata?.name ?? "Empty NFT"}
                    className="size-10 rounded-md object-contain"
                  />
                  <div className={cn("w-full", ellipsableStyles)}>
                    {token.metadata?.name ?? token.token_id}
                  </div>
                </Link>
```
is repeated in multiple part of the codebase and can be placed in a component named `TokenInfosCell`, in `/apps/arkmarket/src/components/cells/token-infos-cell.tsx` for exemple

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by SoarinSkySagar**
Closes #166 

## Checklist
- [x] Created different TableCell components from already existing codebase
- [x] Refactored codebase to use those custom components to follow DRY principle
- [x] Ran the dev and dev commands and ensured no errors

## Screenshot of build command
image

*This PR was created as part of OnlyDust's ODHack 8.0.*

### CODE REVIEW

**Comment by SoarinSkySagar** [comment]
Hi @YohanTz, I have completed all tasks for Issue #166, please review the PR!

File: apps/arkmarket/src/components/cells/activity-time-cell.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { TableCell } from "@ark-market/ui/table";
+import { timeSince } from "@ark-market/ui";
+
+interface ActivityProps {
+   time_stamp: number | null;
```
Comment (YohanTz) [inline apps/arkmarket/src/components/cells/activity-time-cell.tsx:5]: Can you update props names to camelCase? 🙏🏽

File: apps/arkmarket/src/components/cells/activity-up-cell.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default function ActivityUp() {
+    return (
+        <TableCell className="pr-5">
+                <Button asChild size="icon" variant="outline">
+                  <ExternalLink href="/">
+                    <ArrowUpRight className="size-5" />
+                  </ExternalLink>
+                </Button>
+              </TableCell>
```
Comment (YohanTz) [inline apps/arkmarket/src/components/cells/activity-up-cell.tsx:14]: Run prettier on files

**Comment by YohanTz** [comment]
@SoarinSkySagar Thanks for your contribution! I will check your PR during the day

File: apps/arkmarket/src/components/cells/activity-price-cell.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import type { CollectionActivity, PortfolioActivity, PortfolioOffers } from "~/types";
+import { TableCell } from "@ark-market/ui/table";
+
+interface PriceCellProps {
+  activity: CollectionActivity | PortfolioActivity | PortfolioOffers;
+}
+
+export default function PriceCell({ activity }: PriceCellProps) {
+  return (
+    <TableCell className="flex w-[25%]">
```
Comment (YohanTz) [inline apps/arkmarket/src/components/cells/activity-price-cell.tsx:11]: remove `w-[25%]`, it breaks this page for example: `wallet/0x64ae291c8482c273828957239017c3dda0ec12ef27f76c9c9b043740cdd9137?activeTab=activity`

**Comment by SoarinSkySagar** [comment]
GM @YohanTz, thank you for the review; I have made the changes requested by you now, please check.

**Comment by SoarinSkySagar** [comment]
@YohanTz fixed all stuff, please review :)

**Review by YohanTz — APPROVED** [review]
LGTM!

**Comment by YohanTz** [comment]
Can you please solve conflicts so I can merge?

**Review by YohanTz — APPROVED** [review]

**Comment by SoarinSkySagar** [comment]
Thanks, for merging @YohanTz!

> LGTM!

Btw, what does LGTM mean?

**Comment by YohanTz** [comment]
Thank you for your contribution! @SoarinSkySagar

**Pull request merged**